### DISCUSSION TITLE
Error with connecting to https huggingface.co running ingest.py

### DISCUSSION THREAD

**Discussion by HabMichVerlaufen** [imartinez/privateGPT#470] [General]
Hi all, running the ingest.py gives me an Error for the huggingface.co connection.
Does someone know where this might come from or how I can solve this?

`requests.exceptions.SSLError: HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /api/models/sentence-transformers/all-MiniLM-L6-v2 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1002)')))`

I tried with this, but no success:
```
import os
os.environ['CURL_CA_BUNDLE'] = ''
```
Python 3.11.3
Requests 2.31

**Comment by pretend7979** [comment]
I have this issue as well. However it's Python 3.10 for me. This is on a Corp computer. And the Cert is self signed by my company. I believe that's the issue. I installed the huggingface.co cert in the root store. But I get an issue with the cdn still.

requests.exceptions.SSLError: (MaxRetryError("HTTPSConnectionPool(host='cdn-lfs.huggingface.co', port=443)

**Reply by Bob-cheng** [reply]
Have you solved this problem? I think I have the same problem with yours.

**Comment by mjahanbin** [comment]
set TRANSFORMERS_OFFLINE=1
